avro2tf/src/test/resources/configs/avro2tf_config_movielens_rank.json

```diff
@@ -115,6 +115,52 @@
         "dtype": "double",
         "shape": []
       }
+    },
+    {
```

Comment:
The isLabel is only useful for padding scalar values. Can we put one label example that is not 2D, thanks

Reply:
We have the "response" label on lines 109-118. Should I add another?

Reply:
yes, since we are testing for multiple labels, another scalar type label will be good

Reply:
Added "response2", essentially a duplicate of "response"